samples/subsys/smf/smf_calculator/sample.yaml

```diff
@@ -0,0 +1,22 @@
+sample:
```

Comment:
Can someone sanity-check these paths?

Reply:
I don't think it's really a path, just a test identifier and it somehow loosely matches the path, I would use `sample.smf.smf_calculator` (seems coherent with the other sample in `smf`).

The `name` down here is stale though, you probably want to use what you have as description as "name" instead, and I guess you can omit the description entirely, or expand it? Up to you, look for other `sample.yaml` files.